README.md

```diff
@@ -53,25 +54,17 @@ For a more detailed guide, please refer to our
     ./setup.sh
     ```
 
-2.  If Tock OS is already installed on your board, move to the next step.
-    Otherwise, just run one of the following commands, depending on the board
-    you have:
+2.  Next step is to install Tock OS as well as the OpenSK application on your
+    board (**Warning**: it will erase the locally stored credentials). Run:
 
     ```shell
     # Nordic nRF52840-DK board
-    ./deploy.py os --board=nrf52840_dk
+    ./deploy.py --board=nrf52840dk --opensk
```

Comment:
I found it useful to have the possibility of flashing only os or app (to avoid flakiness of the flashing protocol), at least for boards that support it. Only supporting a "flash all" would be a regression.

Reply:
Flashing the app without the OS is possible by passing `--no-tockos` flag.
The scenario that is not possible anymore is flashing only the OS because of DFU